pkgs/development/python-modules/pint/default.nix

```
@@ -25,8 +25,10 @@ buildPythonPackage rec {
   disabled = pythonOlder "3.6";
 
   propagatedBuildInputs = [
+    importlib-metadata
+    packaging
     setuptools_scm
```

should be:
```suggestion
  nativeBuildInputs = [ setuptools_scm ];
  
  propagatedBuildInputs = [ packaging ]
    ++ lib.optionals (pythonOlder "3.8") [ importlib-metadata ];
```
if that doesn't work, then upstream has mispackaged this.